### PR TITLE
Feature/index query

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

File: web/src/pages/index.js [hunk 1]
```diff
@@ -119,12 +117,45 @@ export const query = graphql`
...
+        subHero: allSanityIndexPage {
+            edges {
+                node {
+                    subHero {
+                        heading
+                        info
+                    }
+                }
+            }
+        }
```
Comment (asasmith) [inline web/src/pages/index.js:129]: You shouldn't need to use the `allSanityIndexPage` query for these. Try:
```
  sanityIndexPage {
    subHero {
      heading
      info
    }

    visionSection {
      heading
      info
    }
    ...
  }
```

This should make accessing the properties you need a little easier.

```
const { sanityIndexPage: { subHero, visionSection } } = data;

...

<BgImageWithCard
  data={subHero}
  ...
 >
  ```

**Review by asasmith — CHANGES_REQUESTED** [review]

File: web/src/pages/index.js (hunk 1 above)
Comment (kcbreuil) [reply web/src/pages/index.js:129]: @asasmith ah okay this is so helpful! thank you!

Comment (asasmith) [reply web/src/pages/index.js:129]: @kcbreuil yw. You'll only need to use the `all*` queries if you want to return a collection of pages (like blog posts).

File: web/src/pages/index.js [hunk 2]
```diff
@@ -95,49 +69,81 @@ export default function IndexPage({ data }) {
...
+            }
+            subHero {
+                heading
+                info
+            }
+            whatWeDoSection {
+                heading
+                info
+            }
+        }
```
Comment (asasmith) [inline web/src/pages/index.js:121]: Should each of these sections have a unique image?

File: web/src/components/bgImagewithCard.js [hunk 3]
```diff
@@ -18,15 +21,15 @@ export default function BgImagewithCard({
     to,
     href,
 }) {
-    const {
-        childImageSharp: { fluid },
-    } = image;
+    const { asset } = image;
+    const imageData = getImage(asset);
+    const bgImageData = convertToBgImage(imageData);
```
Comment (asasmith) [inline web/src/components/bgImagewithCard.js:26]: @kcbreuil I think all you need to do is change this to
```
const { backgroundImage: { asset } } = data;
const imageData = getImage(asset);
const bgImageData = convertToBgImage(imageData);
```

**Review by asasmith — APPROVED** [review]

**Pull request merged**